waves-integration/src/main/scala/com/wavesplatform/dex/grpc/integration/WavesBlockchainClientBuilder.scala

```diff
@@ -0,0 +1,33 @@
+package com.wavesplatform.dex.grpc.integration
+
+import com.wavesplatform.dex.grpc.integration.clients.{WavesBlockchainCachingClient, WavesBlockchainClient, WavesBlockchainGrpcAsyncClient}
+import com.wavesplatform.dex.grpc.integration.settings.WavesBlockchainClientSettings
+import com.wavesplatform.utils.ScorexLogging
+import io.grpc.internal.DnsNameResolverProvider
+import io.netty.channel.nio.NioEventLoopGroup
+import monix.execution.Scheduler
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object WavesBlockchainClientBuilder extends ScorexLogging {
+  def async(wavesBlockchainClientSettings: WavesBlockchainClientSettings,
+            monixScheduler: Scheduler,
+            grpcExecutionContext: ExecutionContext): WavesBlockchainClient[Future] = {
+    log.info(s"Building gRPC client for server: ${wavesBlockchainClientSettings.grpc.target}")
+
+    val eventLoopGroup = new NioEventLoopGroup
+    val channel = wavesBlockchainClientSettings.grpc.toNettyChannelBuilder
+      .nameResolverFactory(new DnsNameResolverProvider)
+      .defaultLoadBalancingPolicy("pick_first")
```

Comment:
Actually this parameter is used by default

Reply:
Done